app/src/main/res/layout/list_widget.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
+
+    <LinearLayout
+            android:id="@+id/widget_title_container"
+            android:background="#88000000"
+            android:layout_width="match_parent"
+            android:layout_height="28dp"
+            android:orientation="horizontal">
+
+        <ImageView
+                android:layout_width="20dp"
+                android:layout_height="20dp"
+                android:src="@drawable/ic_launcher"
+                android:layout_margin="4dp"/>
+
+        <TextView
+                android:layout_width="wrap_content"
+                android:layout_gravity="center_vertical"
+                android:textColor="@android:color/white"
+                android:layout_height="wrap_content"
+                android:text="@string/app_name"/>
```

Comment:
Can you use drawableLeft directly in TextView instead of Linear+ImageView+TextView to draw a drawable on the left ?